.env

```diff
@@ -0,0 +1,5 @@
+ALPINE_RUBY_VERSION=2.6.5
```

Comment:
These are default values. All of them are overrideable via environment variable and will be used by the docker images.

Reply:
Cool.